proxies/sandbox/apiproxy/policies/OAuthV2.VerifyAccessToken.xml

```diff
@@ -1,3 +1,4 @@
 <OAuthV2 async="false" continueOnError="false" enabled="false" name="OauthV2.VerifyAccessToken">
   <Operation>VerifyAccessToken</Operation>
+  <Scope>urn:nhsd:apim:app:level3:e-referrals-service-api urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api</Scope>
```

Comment:
As above I think only the AAL3 scope should be being included here as we don't yet want to support application-restricted access.